src/mqtt_client/mqtt_client.rs

```diff
@@ -62,10 +62,20 @@ where
         broker: IpAddr,
         clock: Clock,
     ) -> Result<Self, minimq::Error<Stack::Error>> {
-        let mqtt = minimq::Minimq::new(broker, client_id, stack, clock)?;
-
-        let mut response_topic: String<128> = String::from(prefix);
-        response_topic.push_str("/log").unwrap();
+        let mut mqtt = minimq::Minimq::new(broker, client_id, stack, clock)?;
+
+        // Configure a will so that we can indicate whether or not we are connected.
+        let mut connection_topic: String<75> = String::from(prefix);
+        connection_topic.push_str("/connected").unwrap();
+        mqtt.client
+            .set_will(
+                &connection_topic,
+                "0".as_bytes(),
```

Comment:
Should we standardize this message structure to something a bit more complex for more flexibility in the future without breaking compatibility?

Reply:
I think this is fine as is. It's valid JSON.
Other useful info that is unlike telemetry (like application version, hardware version, boot time, etc. should likely be a different path).